### PR TITLE
[6.5.x] RHBPMS-4084 - Boundary events are not generated for MI and Event Sub-…

### PR DESCRIPTION
**Pull request by tsurdilo**
…processes

### CODE REVIEW

File: jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/impl/Bpmn2UnmarshallingTest.java [hunk 1]
```diff
@@ -1366,4 +1366,42 @@ private void verifyAttribute(BaseElement element, String attributeName, Object a
 
         fail(attributeName + " with value: " + attributeValue + " was not found");
     }
+
+    @Test
+    public void testMIandEventProcessBoundaryEvents() throws Exception {
+        Bpmn2JsonUnmarshaller unmarshaller = new Bpmn2JsonUnmarshaller();
```
Comment (jomarko) [inline jbpm-designer-backend/src/test/java/org/jbpm/designer/bpmn2/impl/Bpmn2UnmarshallingTest.java]: Remove this line please.

**Comment by tsurdilo** [comment]
@jomarko amended - thanks

**Comment by tsurdilo** [comment]
resolved branch conflict

**Comment by jomarko** [comment]
+1 if jenkins pass

**Pull request merged**